<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialogBase</class>
 <widget class="QDialog" name="SettingsDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout">
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="m_pGroupAdvanced">
     <property name="title">
      <string>&amp;Advanced</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="m_pLabel_19">
        <property name="text">
         <string>Sc&amp;reen name:</string>
        </property>
        <property name="buddy">
         <cstring>m_pLineEditScreenName</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="m_pLineEditScreenName">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="m_pLabel_20">
        <property name="text">
         <string>P&amp;ort:</string>
        </property>
        <property name="buddy">
         <cstring>m_pSpinBoxGamePoll</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="m_pLabel_21">
        <property name="text">
         <string>&amp;Interface:</string>
        </property>
        <property name="buddy">
         <cstring>m_pLineEditInterface</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="m_pLineEditInterface">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="m_pSpinBoxPort">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>24800</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0" rowspan="2" colspan="2">
    <widget class="QGroupBox" name="m_pGroupLog">
     <property name="title">
      <string>Logging</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="m_pLabel_3">
        <property name="text">
         <string>&amp;Logging level:</string>
        </property>
        <property name="buddy">
         <cstring>m_pComboLogLevel</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="m_pCheckBoxLogToFile">
        <property name="text">
         <string>Log to file:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="m_pLineEditLogFilename">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="m_pButtonBrowseLog">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="m_pComboLogLevel">
        <item>
         <property name="text">
          <string>Error</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Warning</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Note</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Info</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Debug</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Debug1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Debug2</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="m_pGroupGameDevices">
     <property name="title">
      <string>&amp;Game device support</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="m_pLabel_22">
        <property name="text">
         <string>&amp;Mode:</string>
        </property>
        <property name="buddy">
         <cstring>m_pLineEditInterface</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="m_pComboBoxGameDevice">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Next Generation (XInput) - 32-bit only</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Legacy (JOYINFOEX)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="m_pLabel_23">
        <property name="text">
         <string>&amp;Polling:</string>
        </property>
        <property name="buddy">
         <cstring>m_pLineEditInterface</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QRadioButton" name="m_pRadioButtonGamePollDynamic">
        <property name="text">
         <string>Dynamic (align with client polling)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="m_pRadioButtonGamePollStatic">
          <property name="text">
           <string>Static frequency:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="m_pSpinBoxGamePoll">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>60</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="m_pGroupStart">
     <property name="title">
      <string>&amp;Startup</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QCheckBox" name="m_pCheckBoxAutoStart">
        <property name="text">
         <string>&amp;Start Synergy after logging in</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_pCheckBoxAutoConnect">
        <property name="text">
         <string>&amp;Automatically start server/client</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_pCheckBoxAutoHide">
        <property name="text">
         <string>&amp;Hide when server/client starts</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_pLineEditScreenName</tabstop>
  <tabstop>m_pLineEditInterface</tabstop>
  <tabstop>m_pCheckBoxAutoConnect</tabstop>
  <tabstop>m_pComboLogLevel</tabstop>
  <tabstop>m_pCheckBoxLogToFile</tabstop>
  <tabstop>m_pLineEditLogFilename</tabstop>
  <tabstop>m_pButtonBrowseLog</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
